code/game/gamemodes/cult/cult_comms.dm

```diff
@@ -65,3 +69,144 @@
+								new /obj/effect/overlay/temp/cult/blood(final)
+								addtimer(CALLBACK(M, /mob/living.proc/reckon, final), 10)
+							else
+								for(var/datum/action/innate/cultmast/finalreck/H in owner.actions)
+									qdel(H)
+
+/mob/living/proc/reckon(var/turf/final)
+	new /obj/effect/overlay/temp/cult/blood/out(get_turf(src))
+	forceMove(final)
+
+
+
+/datum/action/innate/cultmast/cultmark
+	name = "Mark Target"
+	desc = "Marks a target for the cult"
+	button_icon_state = "cult_mark"
+	background_icon_state = "bg_demon"
+	buttontooltipstyle = "cult"
+	check_flags = AB_CHECK_RESTRAINED|AB_CHECK_STUNNED|AB_CHECK_CONSCIOUS
+	var/obj/effect/proc_holder/cultmark/CM
+	var/time = 0
+
+/datum/action/innate/cultmast/cultmark/New()
+    CM = new()
+    ..()
+
+/datum/action/innate/cultmast/cultmark/IsAvailable()
+	if(owner.mind.special_role != "Cult Master")
+		return 0
+	if((world.time - time)<900)
+		owner << "<span class='cultlarge'><b>You need to wait [round((900-(world.time-time))/10)] seconds before you can mark another target!</b></span>"
+		return 0
+	return ..()
+
+/datum/action/innate/cultmast/cultmark/Destroy()
+    qdel(CM)
+    CM = null
+    return ..()
+
+/datum/action/innate/cultmast/cultmark/Activate()
+    CM.toggle(owner) //the important bit
+    time = world.time
+    return TRUE
+
+/obj/effect/proc_holder/cultmark
+    active = FALSE
+    ranged_mousepointer = 'icons/effects/cult_target.dmi'
+
+
+/obj/effect/proc_holder/cultmark/proc/toggle(mob/user)
+    if(active)
+        remove_ranged_ability("You cease the marking ritual...")
+    else
+        add_ranged_ability(user, "You prepare to mark a target for your cult...")
+
+/obj/effect/proc_holder/cultmark/InterceptClickOn(mob/living/caller, params, atom/target)
+	if(..())
+		return
+	if(ranged_ability_user.incapacitated())
+		remove_ranged_ability()
+		return
+	var/turf/T = ranged_ability_user.loc
+	if(!isturf(T))
+		return FALSE
+	if(target in view(7, get_turf(ranged_ability_user)))
+		GLOB.blood_target = target
+		for(var/datum/mind/M in SSticker.mode.cult)
+			M.current << "<span class='cultlarge'><b>Master [ranged_ability_user] has marked [GLOB.blood_target] as the cult's highest priority, get there immediately!</b></span>"
+			M.current << pick(sound('sound/hallucinations/over_here2.ogg',0,1,75), sound('sound/hallucinations/over_here3.ogg',0,1,75))
+			new /obj/effect/cultmark(M, GLOB.blood_target)
+		remove_ranged_ability(caller, "The marking rite is complete! It will last for one minute.")
+		addtimer(CALLBACK(GLOBAL_PROC, .proc/reset_blood_target), 1200, TIMER_OVERRIDE)
+		return TRUE
+	return FALSE
+
+/proc/reset_blood_target()
+	for(var/datum/mind/M in SSticker.mode.cult)
+		M.current << "<span class='cultlarge'><b>The blood mark on [GLOB.blood_target] has expired!</b></span>"
+	GLOB.blood_target = null
+
+
+obj/effect/cultmark
+	var/image/cult_marker
+	var/mob/living/viewer
+
+obj/effect/cultmark/Initialize(var/mob/living/T, var/atom/BT)
```

Comment:
I actually had mapload in there and the first argument was the mark's turf - but removed it to see if thats what was preventing the image from appearing. That wasn't the case.